stacks-signer/src/runloop.rs

```diff
@@ -873,6 +1004,9 @@ impl From<&Config> for RunLoop<FireCoordinator<v2::Aggregator>> {
             signer_ids: config.signer_ids.clone(),
             signer_id: config.signer_id,
             signer_addresses: vec![],
+            nack_messages_received: HashMap::new(),
+            nack_messages_sent: HashSet::new(),
+            nack_threshold: (total_signers + 1) / 2,
```

Comment:
This should be a `get_nack_threshold` function, which returns the threshold based off of `coordinator.total_signers`. That way when we later have code to update `coordinator` for new reward cycles, we don't have to manually update the nack threshold too.